packages/shylu/shylu_node/tacho/example/Tacho_ExampleDriver.hpp

```diff
@@ -271,13 +265,20 @@ template <typename value_type> int driver(int argc, char *argv[]) {
         t("t", A.NumRows(), nrhs);                  // temp workspace (store permuted rhs)
 
     {
-      if (onesRHS) {
+      if (rhs_file.length() > 0) {
+        if(Tacho::MatrixMarket<value_type>::readDenseVectors(rhs_file, b, verbose) != 0) {
+          return -1;
+        }
```

Comment:
For consistency with the other options, would it help to add a verbose output indicating a matrix market file was read for the rhs, maybe emitting the rhs_file string?

Reply:
I print the info inside `readDenseVectors` function.